usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -422,17 +429,22 @@ def add_arguments(self, parser):
             "--id-file",
             metavar="FILEPATH",
             type=str,
-            help="A file containing only transaction IDs (detached_award_procurement_id) "
+            help="(OPTIONAL) A file containing only transaction IDs (detached_award_procurement_id) "
             "to reload, one ID per line. Nonexistent IDs will be ignored.",
         )
 
     @transaction.atomic
     def handle(self, *args, **options):
-        if any([options["detached_award_procurement_ids"], options["id_file"]]):
-            ids_from_file = read_afa_ids_from_file(options["id_file"]) if options["id_file"] else set()
-            explicit_ids = set(options["detached_award_procurement_ids"]) if options["detached_award_procurement_ids"] else set()
-            detached_award_procurement_ids = list(explicit_ids | ids_from_file)
+        if options["id_file"]:
+            self.load_fpds_from_file(options["id_file"])
+
+        if options["detached_award_procurement_ids"]:
```

Comment:
Is it intentional to allow a file to be read and loaded in conjunction with a parameter list of IDs?

Reply:
It seemed to not do any harm, and in theory someone might want to do it. This functionality was tested.